yarn-project/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -83,6 +88,91 @@ contract Test {
         context.this_address()
     }
 
+    #[aztec(private)]
+    fn call_create_note(
+        value: Field,
+        owner: AztecAddress,
+        storage_slot: Field
+    ) {
+        assert(storage_slot != 1, "storage slot 1 is reserved for example_constant");
+
+        let mut note = ValueNote::new(value, owner);
+        create_note(
+            &mut context,
+            storage_slot,
+            &mut note,
+            ValueNoteMethods,
+            true,
+        );
+    }
+
+    #[aztec(private)]
+    fn call_get_notes(
+        storage_slot: Field,
+        includeNullified: bool,
+    ) {
+        assert(storage_slot != 1, "storage slot 1 is reserved for example_constant");
+
+        let mut options = NoteGetterOptions::new();
+        if (includeNullified) {
+            options = options.include_nullified();
+        }
+
+        let opt_notes = get_notes(
+            &mut context,
+            storage_slot,
+            ValueNoteMethods,
+            options,
+        );
+
+        // We can't get the return value of a private function from the outside world in an end to end test, so we
+        // expose it via an unecrypted log instead.
+        let value = opt_notes[0].unwrap().value;
+        emit_unencrypted_log_from_private(&mut context, value);
+    }
+
+    unconstrained
```

Comment:
Is this formatted? Unconstrained is not a macro, so we are just putting it directly on the same line.

Reply:
It does look wrong, I'll check why the noir code is not being autoformatted.